craft_application/app.py

```diff
@@ -155,22 +255,28 @@ def run(self) -> int:
             emit.ended_ok()
         except KeyboardInterrupt as err:
             self._emit_error(CraftError("Interrupted."), cause=err)
-            retcode = 1
+            retcode = 2
```

Comment:
out of curiosity, where does this `2` come from?

Reply:
I messed it up, it's supposed to be 130 (128+2).

When a process receives a signal it's standard to return a status code of `128 + n` where `n` is the signal number. `SIGINT` is signal 2 pretty much across the board, so it should be 130. An uncaught `KeyboardInterrupt` in Python will return status code 130.

I don't have a *great* reference for this, but [`sysexits.h`](https://manpages.ubuntu.com/manpages/lunar/man3/sysexits.h.3head.html) and [TLDP's advanced bash scripting](https://tldp.org/LDP/abs/html/exitcodes.html) are what I tend to reference. There's no real standard AFAICT though, so if there's a company or team standard that should take precedence